### PR TITLE
Red pills sometimes make people think

### PR DESCRIPTION
**Pull request by 81Denton**
Remember that scene from The Matrix? This PR adds a 5% chance that  swallowing red pills makes you think about stuff the average Nanotrasen employee isn't supposed to pay attention to.

### CODE REVIEW

**Comment by adamkad1** [comment]
what about blue pills?-

**Comment by optimumtact** [comment]
use the json strings system

**Comment by 81Denton** [comment]
@adamkad1 There sadly aren't any sprites for plain blue pills

**Comment by Anonmare** [comment]
Why does NT hire clowns and mimes for every station?
Why is the air supply hooked up to the plasma tank?
Why are there fire alarms but no sprinklers?
Why is this a plasma research station if we know everything about plasma already?

**Comment by IllotAFV** [comment]
There is a problem with emergency shuttle. By LORE every single game is the same shift and date, but it just rewinded back in time. So, technically every new game is not a new shift, it’s the old one, when code blue happened.

**Comment by kingoftheautism** [comment]
Why we have gateway if we never can activate it anyways?
What the hell happened with thermodynamic laws after inventing thermo-electric generator?

**Comment by GuyonBroadway** [comment]
@IllotAFV the lore of what shift is what doesn't make any sense because the game is a self contradictory mess at the best of times, and that's probably because ss13 is the proverbial horse built by committee and everyone who takes a stab at the lore has thier own ideas on how it should be.


To get back on topic

"Why does a top secret deep space research facility need a bar and restaurant?"

"Why does a top secret deep space research facility have a brig the size of a large county jail?"
"How come a hole in the floor doesn't suck you out into space?"

**Comment by IllotAFV** [comment]
@GuyonBroadway if this doesnt work like that,what is the point of this super cool research facility? If it's being attacked by everyone every shift possible. NT could have powered up their defences out there or something.

**Comment by GuyonBroadway** [comment]
Because ss13 is a collaborative open source video game with each person who adds to the lore having thier own idea of how it should be.

**Comment by XDTM** [comment]
Why is space cold?

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -46,6 +47,9 @@
 		M.visible_message("<span class='danger'>[user] forces [M] to [apply_method] [src].</span>", \
 							"<span class='userdanger'>[user] forces [M] to [apply_method] [src].</span>")
 
+	makes_me_think = pick(strings("redpill.json", "redpill_questions"))
+	if(icon_state == "pill4" && prob(1)) //you take the red pill - you stay in Wonderland, and I show you how deep the rabbit hole goes
```
Comment (XDTM) [inline code/modules/reagents/reagent_containers/pill.dm]: why not make makes_me_think a var here instead of on the pill itself?

**Comment by Time-Green** [comment]
Might not hurt to make them say it out loud
Also
Why does it look like ever star is 5 meter away

**Comment by GuyonBroadway** [comment]
Talking out loud isn't thinking.

File: code/modules/reagents/reagent_containers/pill.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/reagents/reagent_containers/pill.dm]: Do you mean like this?
`var/makes_me_think = pick(strings("redpill.json", "redpill_questions"))`

**Comment by 81Denton** [comment]
Thanks for all the ideas, I'll add them once I have some free time btw

File: code/modules/reagents/reagent_containers/pill.dm (hunk 1 above)
Comment (XDTM) [reply code/modules/reagents/reagent_containers/pill.dm]: yup

**Comment by Ratvar** [comment]
"Traps are gay."

**Comment by fluffe9911** [comment]
Why does space circle around on itself?

**Comment by 81Denton** [comment]
Okay, I added a bunch. Also increased the chance to 5%, since a 1% chance is realistically too low for them to ever show.

**Comment by Swiftkor** [comment]
"Where did the ceiling go?"

**Comment by ExcessiveUseOfCobblestone** [comment]
brain damage lines part 2

**Comment by Ratvar** [comment]
should add something about *braindamage* lines, emoji, caps and netspeak
and dumb "made me think" parody into braindamage for cycle

**Comment by Qustinnus** [comment]
wow fucking meme thief I did this before it was cool https://github.com/tgstation/tgstation/pull/41825/files#diff-da9abd3d87b55aa6a86c0be5c38ca16cR250

hello meme police??

**Pull request merged**